### PR TITLE
2 of 4, WIP ci: ds-661 dependabot groups for bulk updates

### PR DESCRIPTION
**Pull request by cdcabrera**
## What's included

- ci: ds-661 dependabot groups for bulk updates

### Notes
- moves legacy v1 temporarily over to dependabot groups to help with bulk updates
   - bulk updates of NPM packages are broken out by semver to help alleviate questions around packages and "[will this break the UI?]"
      - individual packages that have MAJOR updates will receive independent PRs
      - `Dev-Dependency`packages that have MINOR and PATCH updates will receive bulk/grouped PRs
      - `Dependency`packages that have PATCH updates will receive bulk/grouped PRs
      - `Dependency`packages that have MINOR updates will receive bulk/grouped PRs
- v2 will leverage a similar update
- @ruda the quipudocs work we'll resolve with #347

## How to test
... Trust dependabot, dependabot knows ALL... nothing to test specifically we leveraged existing, working, dependabot updates to other UI codebases to do this work
   - https://github.com/RedHatInsights/hybrid-committed-spend-ui/blob/main/.github/dependabot.yml
   - https://github.com/RedHatInsights/curiosity-frontend/blob/main/.github/dependabot.yml

## Example

...

## Updates issue/story

[DISCOVERY-661](https://issues.redhat.com/browse/DISCOVERY-661)
blocked by #345

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -263,8 +198,8 @@ This is the slowest part of package updates. If any packages are skipped during
 
 Before developing you'll need to install:
 * [NodeJS and NPM](https://nodejs.org/)
-* [Docker](https://docs.docker.com/desktop/)
-  * Alternatively, you can try [Podman](https://github.com/containers/podman). [Homebrew](https://brew.sh/) can be used for the install `$ brew install podman`
+* [podman desktop](https://podman-desktop.io/)
+  * Alternatively, you can try [Docker](https://docs.docker.com/desktop/).
```
Comment (infinitewarp) [inline CONTRIBUTING.md:202]:
```suggestion
```

No support anywhere for Docker Desktop. 🚫

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -229,8 +164,8 @@ This is the slowest part of package updates. If any packages are skipped during
 1. After each package version update in [`package.json`](./package.json) you'll run the follow scripts
    - `$ yarn test`, if it fails you'll need to run `$ yarn test:dev` and update the related tests
    - `$ yarn build`, if it fails you'll need to run `$ yarn test:integration-dev` and update the related tests
-   - Make sure Docker/Podman is running, then type `$ yarn start`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
-   - Make sure Docker/Podman is running, then type `$ yarn start:stage`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
+   - Make sure podman desktop, or Docker, is running, then type `$ yarn start`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
+   - Make sure podman desktop, or Docker, is running, then type `$ yarn start:stage`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
```
Comment (infinitewarp) [inline CONTRIBUTING.md:168]:
```suggestion
   - Make sure podman desktop is running, then type `$ yarn start`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
   - Make sure podman desktop is running, then type `$ yarn start:stage`. Confirm that staging run is still accessible and that no design alterations have happened. Fix accordingly.
```

No support anywhere for Docker Desktop. 🚫

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -398,7 +318,7 @@ This is a local run designed to function with minimal resources and a mock API.
     ```
     REACT_APP_DEBUG_MIDDLEWARE=true
     ```
-1. Make sure Docker/Podman is running
+1. Make sure podman desktop, or Docker, is running
```
Comment (infinitewarp) [inline CONTRIBUTING.md:321]:
```suggestion
1. Make sure podman desktop is running
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -420,7 +340,7 @@ This is an authenticated local run that has the ability to run against a contain
...
```
Comment (infinitewarp) [inline CONTRIBUTING.md:343]:
```suggestion
1. Make sure podman desktop is running
```

File: README.md [hunk 5]
```diff
@@ -10,8 +10,8 @@ Web user interface for [Quipucords](https://github.com/quipucords/quipucords), b
 ### Requirements
 Before developing, the basic requirements:
  * Your system needs to be running [NodeJS version 18+ and NPM](https://nodejs.org/)
- * [Docker](https://docs.docker.com/desktop/)
-   * Alternatively, you can try [Podman](https://github.com/containers/podman).
+ * [podman desktop](https://podman-desktop.io/)
+   * Alternatively, you can try [Docker](https://docs.docker.com/desktop/).
```
Comment (infinitewarp) [inline README.md:14]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ This is the base context for running a local UI against a mock API.
     ```
     REACT_APP_DEBUG_MIDDLEWARE=true
    ```
-1. Make sure Docker/Podman is running
+1. Make sure podman desktop, or Docker, is running
```
Comment (infinitewarp) [inline README.md:37]:
```suggestion
1. Make sure podman desktop is running
```

File: scripts/apiStage.sh [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+        p ) PORT=$OPTARG;;
+        t ) TYPE="$OPTARG";;
+      esac
+  done
+
+  if [ "$(command -v podman)" ]; then
+    PODMAN="podman"
+  elif [ "$(command -v PODMAN)" ]; then
+    PODMAN="docker"
+  fi
```
Comment (infinitewarp) [inline scripts/apiStage.sh:121]:
```suggestion
  PODMAN="$(command -v podman)"
```

File: scripts/apiStage.sh [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+  if [ -z "$PODMAN" ]; then
+    printf "${RED}Missing container tooling. Unable to set \$PODMAN. podman, or docker, is required.${NOCOLOR}\n"
```
Comment (infinitewarp) [inline scripts/apiStage.sh:124]:
```suggestion
    printf "${RED}Missing container tooling. Unable to set \$PODMAN. podman is required.${NOCOLOR}\n"
```

File: scripts/apiStage.sh [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+    printf "\n\n${NOCOLOR}"
+  else
+    printf "${RED}Container ERROR"
+    printf "\n\n  Error: ${RED}Check container \"${CHECKONE}\""
+    printf "${NOCOLOR}\n"
+  fi
+}
+#
+#
+# Clean podman/docker
```
Comment (infinitewarp) [inline scripts/apiStage.sh:34]:
```suggestion
# Clean podman```

File: CONTRIBUTING.md (hunk 1 above)
Comment (cdcabrera) [reply CONTRIBUTING.md:202]: k, this PR is 2 of 4... the podman/docker work is here #345... I'll apply the comments and remove docker completely

Comment (cdcabrera) [reply CONTRIBUTING.md:202]: see #345 for resolve

File: CONTRIBUTING.md (hunk 2 above)
Comment (cdcabrera) [reply CONTRIBUTING.md:168]: see #345 for resolve

File: CONTRIBUTING.md (hunk 3 above)
Comment (cdcabrera) [reply CONTRIBUTING.md:321]: see #345 for resolve

File: CONTRIBUTING.md (hunk 4 above)
Comment (cdcabrera) [reply CONTRIBUTING.md:343]: see #345 for resolve

File: README.md (hunk 5 above)
Comment (cdcabrera) [reply README.md:14]: see #345 for resolve

File: README.md (hunk 6 above)
Comment (cdcabrera) [reply README.md:37]: see #345 for resolve

File: scripts/apiStage.sh (hunk 7 above)
Comment (cdcabrera) [reply scripts/apiStage.sh:121]: see #345 for resolve

**Review by bruno-fs — APPROVED** [review]

**Review by nicolearagao — APPROVED** [review]
LGTM

**Pull request merged**